### DISCUSSION TITLE
Some dependencies not notified

### DISCUSSION THREAD

**Discussion by jancama2** [renovatebot/renovate#9044] [Request Help] [answered]
Hi,
I have this simple config:
```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "enabledManagers": ["npm", "nvm", "nodeenv", "gitlabci-include", "dockerfile", "regex"],
  "commitMessagePrefix": "⬆️",
}
```
but Renovate doesn't seem to notify all the dependencies. There are certain such as:
```
"connected-react-router": "^6.7.0",
"date-fns": "^2.11.1",
"fela": "^11.2.0",
"formik": "^2.1.4",
"tinycolor2": "^1.4.1",
"grpc-tools": "^1.8.1",
```
and many more that I haven't got any notification for even though there are new minor and patch versions of those.
I was thinking about missing `yarn` package manager but not sure. 

Have anyone experienced the same problems? Thx a lot!

**Comment by viceice** [comment]
please check the debug logs

**Comment by rarkins** [comment, marked as answer]
Most likely issue is that you need to switch [rangeStrategy](https://docs.renovatebot.com/configuration-options/#rangestrategy) from `replace` to `update-lockfile`. By default Renovate won't update if the new version satisfies the existing range in `package.json`.

**Reply by jancama2** [reply]
@rarkins you are right, the problem was I didn't know about the strategies, thanks :))
